newsletter-january-2022.rst

```diff
@@ -0,0 +1,88 @@
+.. post:: January 12, 2022
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in January.
+
+Read the Docs newsletter - January 2022
+=======================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We are now managing custom domains for our corporate users using Cloudflare SSL for SaaS,
+  which will remove the manual work that was needed on our side
+  and make the process of setting up a custom domain almost instantaneous.
+
+New features
+------------
+
+December and January have been slow months,
+and we mainly worked on internal changes and documentation improvements.
+Among the major user-facing changes:
+
+- We splitted the developer documentation from the user documentation to avoid confusion
+  and made our :doc:`Read the Docs for Business docs <readthedocs:commercial/index>` more visible.
+- We expanded our user documentation with more examples on how to use MyST Markdown
+  and a :doc:`guide to migrate from reST to MyST <readthedocs:guides/migrate-rest-myst>`
+  (:doc:`more about our committment to support MyST in our blog </sphinx-markdown-2021>`).
+
+Thanks to our external contributor `Çağatay Yiğit Şahin`_.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs
+Changelog <readthedocs:changelog>`.
+
+.. _Çağatay Yiğit Şahin: https://github.com/cagatay-y
+
+Upcoming features
+-----------------
+
+- Ana_ will continue working on the new landing pages,
+  polishing the copywriting along with `Juan Luis`_
+  and translating the mockups to Pelican templates.
+- Anthony_ (TBC)
+- Eric_ (TBC)
+- `Juan Luis`_ will work with Ana_ in the new landing pages,
+  and with Manuel_ in a new process to migrate users from legacy
+  or deprecated configurations.
+- Manuel_ will wrap up the migration to Django 3,
+  improve the queryability of the project configuration in our database,
+  and refactor our task queue to improve its robustness. 
+- Santos_ will add support for multiple ``.readthedocs.yaml`` files per repository.
+
+Possible issues
+---------------
+
+The Python packaging ecosystem is evolving rapidly
+to accomodate for modern standards that will make our life easier,
+and in particular setuptools has been introducing some breaking changes in recent times.
+We anticipated this sort of breakage for our users
+so `we pinned the setuptools version in late
+November <https://github.com/readthedocs/readthedocs.org/pull/8711>`_,
+however a small number of projects did not have this version cap active
```

Comment:
```suggestion
November <https://github.com/readthedocs/readthedocs.org/pull/8711>`_.
However, a small number of projects did not have this version cap active
```

Not sure about this suggestion, but I think it's correct.

Reply:
Yea, it's a long sentence otherwise. 👍